From 7f8c9617a75aeb3a9e1912162f875778ed1de591 Mon Sep 17 00:00:00 2001
From: Nicolas Chauvet <kwizart@gmail.com>
Date: Thu, 1 Feb 2024 12:39:23 +0100
Subject: [PATCH] Add missing include

---
 src/h264.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/h264.c b/src/h264.c
index f3f08b0c0b44..23303672f3a3 100644
--- a/src/h264.c
+++ b/src/h264.c
@@ -36,6 +36,7 @@
 
 #include "request.h"
 #include "surface.h"
+#include "utils.h"
 #include "v4l2.h"
 
 enum h264_slice_type {
-- 
2.42.0

