From 79ff2e8248184f7c2ece817576532d6433aad150 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Wed, 22 May 2024 06:08:53 +0800
Subject: [PATCH] Update h265.c

---
 src/h265.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/h265.c b/src/h265.c
index 4650a51..821c5d5 100644
--- a/src/h265.c
+++ b/src/h265.c
@@ -36,7 +36,7 @@
 #include <sys/mman.h>
 
 #include <linux/videodev2.h>
-#include <hevc-ctrls.h>
+#include <linux/v4l2-controls.h>
 
 #include "v4l2.h"
 
