From 6f346821062eb932fb53c85303c1e61e0d74d2be Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Wed, 22 May 2024 06:39:24 +0800
Subject: [PATCH] Update context.c

---
 src/context.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/src/context.c b/src/context.c
index 04ba9a6..12639be 100644
--- a/src/context.c
+++ b/src/context.c
@@ -40,8 +40,7 @@
 #include <linux/videodev2.h>
 
 #include <mpeg2-ctrls.h>
-#include <h264-ctrls.h>
-#include <hevc-ctrls.h>
+#include <linux/v4l2-controls.h>
 
 #include "utils.h"
 #include "v4l2.h"
@@ -104,7 +103,7 @@ VAStatus RequestCreateContext(VADriverContextP context, VAConfigID config_id,
 	case VAProfileH264ConstrainedBaseline:
 	case VAProfileH264MultiviewHigh:
 	case VAProfileH264StereoHigh:
-		pixelformat = V4L2_PIX_FMT_H264_SLICE_RAW;
+		pixelformat = V4L2_PIX_FMT_H264_SLICE;
 		break;
 
 	case VAProfileHEVCMain:

