From e5c5aa9104f88537a50e7b2ad8a524cbea94971b Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Tue, 21 May 2024 22:01:30 +0800
Subject: [PATCH] Update config.c

---
 src/config.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/config.c b/src/config.c
index e396268..8fcff10 100644
--- a/src/config.c
+++ b/src/config.c
@@ -35,7 +35,7 @@
 #include <linux/videodev2.h>
 
 #include <mpeg2-ctrls.h>
-#include <h264-ctrls.h>
+
 #include <hevc-ctrls.h>
 
 #include "utils.h"
