--- a/lib/wmchat/CMakeLists.txt
+++ b/lib/wmchat/CMakeLists.txt
@@ -4,9 +4,10 @@ project(libwmchat LANGUAGES CXX)
 set(CMAKE_CXX_STANDARD 14)
 
 # Ccache
-find_program(CCACHE_PROGRAM ccache)
-if(CCACHE_PROGRAM)
-  set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${CCACHE_PROGRAM}")
+find_program(TARGET_CC_NOCACHE)
+if(TARGET_CC_NOCACHE)
+  set(CMAKE_C_COMPILER ${TARGET_CC_NOCACHE})
+  set(CMAKE_CXX_COMPILER ${TARGET_CXX_NOCACHE})
 endif()
 
 # Library
